src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -678,276 +678,266 @@ private static class HtmlEntities
-                "\x201a-sbquo",
-                "\x201c-ldquo",
-                "\x201d-rdquo",
-                "\x201e-bdquo",
-                "\x2020-dagger",
-                "\x2021-Dagger",
-                "\x2022-bull",
-                "\x2026-hellip",
-                "\x2030-permil",
-                "\x2032-prime",
-                "\x2033-Prime",
-                "\x2039-lsaquo",
-                "\x203a-rsaquo",
-                "\x203e-oline",
-                "\x2044-frasl",
-                "\x20ac-euro",
-                "\x2111-image",
-                "\x2118-weierp",
-                "\x211c-real",
-                "\x2122-trade",
-                "\x2135-alefsym",
-                "\x2190-larr",
-                "\x2191-uarr",
-                "\x2192-rarr",
-                "\x2193-darr",
-                "\x2194-harr",
-                "\x21b5-crarr",
-                "\x21d0-lArr",
-                "\x21d1-uArr",
-                "\x21d2-rArr",
-                "\x21d3-dArr",
-                "\x21d4-hArr",
-                "\x2200-forall",
-                "\x2202-part",
-                "\x2203-exist",
-                "\x2205-empty",
-                "\x2207-nabla",
-                "\x2208-isin",
-                "\x2209-notin",
-                "\x220b-ni",
-                "\x220f-prod",
-                "\x2211-sum",
-                "\x2212-minus",
-                "\x2217-lowast",
-                "\x221a-radic",
-                "\x221d-prop",
-                "\x221e-infin",
-                "\x2220-ang",
-                "\x2227-and",
-                "\x2228-or",
-                "\x2229-cap",
-                "\x222a-cup",
-                "\x222b-int",
-                "\x2234-there4",
-                "\x223c-sim",
-                "\x2245-cong",
-                "\x2248-asymp",
-                "\x2260-ne",
-                "\x2261-equiv",
-                "\x2264-le",
-                "\x2265-ge",
-                "\x2282-sub",
-                "\x2283-sup",
-                "\x2284-nsub",
-                "\x2286-sube",
-                "\x2287-supe",
-                "\x2295-oplus",
-                "\x2297-otimes",
-                "\x22a5-perp",
-                "\x22c5-sdot",
-                "\x2308-lceil",
-                "\x2309-rceil",
-                "\x230a-lfloor",
-                "\x230b-rfloor",
-                "\x2329-lang",
-                "\x232a-rang",
-                "\x25ca-loz",
-                "\x2660-spades",
-                "\x2663-clubs",
-                "\x2665-hearts",
-                "\x2666-diams",
-            };
-
-            private static LowLevelDictionary<string, char> s_lookupTable = GenerateLookupTable();
-
-            private static LowLevelDictionary<string, char> GenerateLookupTable()
-            {
-                // e[0] is unicode char, e[1] is '-', e[2+] is entity string
+            private const int Count = 253;
```

Comment:
Consider adding a `DEBUG`-only assert to make sure the constant matches the actual count added, something like:

```csharp
#if DEBUG
        static HtmlEntities()
        {
            // Make sure the initial capacity for s_lookupTable is correct
            Debug.Assert(s_lookupTable.Count == Count, "s_lookupTable's initial capacity is incorrect.");
        }
#endif
```

Example:
https://github.com/dotnet/corefx/blob/ac67ffac987d0c27236c4a6cf1255c2bcbc7fe7d/src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs#L418-L423